tests/system/common.py

```diff
@@ -333,9 +334,8 @@ def get_marathon_leader_not_on_master_leader_node():
     if marathon_leader == master_leader:
         delete_marathon_path('v2/leader')
         shakedown.wait_for_service_endpoint('marathon', timedelta(minutes=5).total_seconds())
```

Comment:
If you want to be explicit and test admin router *and* mesosDNS I would make it explicit. Smth. along the lines of:
```
// Check if changes propagated to admin router
shakedown.wait_for_service_endpoint('marathon', timedelta(minutes=5).total_seconds())
with service_endpoint:
   GET /v2/leader
   assert leader != old_leader
// Check mesosDNS too:
marathon_leadership_changed(marathon_leader)
```

Reply:
done